features/step_definitions/assertion_steps.rb

```diff
@@ -3,11 +3,15 @@
 end
 
 Then("I should see {string} in {string}") do |session, status|
-	within("##{status}") do
+	within("#{status}") do 
 		expect(page).to have_content session
 	end
 end
 
 Then("I should not see {string}") do |content|
     expect(page).to have_no_content content
 end
+
```

Comment:
Remove this we are not using it anymore

Reply:
We are using "I should not see" in the login features

Reply:
I meant the code from line 15:
```
Then("I should be on sessions page") do
	expect(current_path).to eq sessions_path
end
```

Reply:
done